indico/web/templates/forms/date_widget.html

```diff
@@ -1,12 +1,32 @@
 {% extends 'forms/base_widget.html' %}
 
 {% block html %}
-    <input type="text" id="{{ field.id }}" value="{{ field._value() }}" name="{{ field.name }}"
-           placeholder="DD/MM/YYYY" {{ input_args | html_params }}>
+    <div class="datetime-widget i-form-field-fixed-width">
```

Comment:
![image](https://user-images.githubusercontent.com/179599/87577582-6bf77500-c6d3-11ea-8b6e-64de16a9d3ad.png)

There's some weird spacing before the clear button. Seems to be coming from `.datetime-widget .clear-pickers`. This fixes it:

```patch
diff --git a/indico/web/client/styles/partials/_inputs.scss b/indico/web/client/styles/partials/_inputs.scss
index 364df0f6e8..66f06f4efc 100644
--- a/indico/web/client/styles/partials/_inputs.scss
+++ b/indico/web/client/styles/partials/_inputs.scss
@@ -276,7 +276,8 @@ $dropdown-transition: $dropdown-transition-step ease-out;
   }
 }

-.datetime-widget {
+.datetime-widget,
+.date-widget {
   .rc-time-picker-input {
     width: 100% !important;
     font-size: 1em;
@@ -348,9 +349,18 @@ $dropdown-transition: $dropdown-transition-step ease-out;
   }
 }

-.i-form .form-group .form-field .datetime-widget input[type='text'] {
-  font-size: 1em;
-  width: 150px;
+.date-widget .clear-pickers {
+  left: 0;
+}
+
+.i-form .form-group .form-field {
+  .datetime-widget,
+  .date-widget {
+    input[type='text'] {
+      font-size: 1em;
+      width: 150px;
+    }
+  }
 }

 .has-warning {
diff --git a/indico/web/templates/forms/date_widget.html b/indico/web/templates/forms/date_widget.html
index 1107fcb8b7..403f88ce87 100644
--- a/indico/web/templates/forms/date_widget.html
+++ b/indico/web/templates/forms/date_widget.html
@@ -1,10 +1,11 @@
 {% extends 'forms/base_widget.html' %}

 {% block html %}
-    <div class="datetime-widget i-form-field-fixed-width">
+    <div class="date-widget i-form-field-fixed-width">
         {# the hidden field has autofocus on purpose to prevent our JS from focusing the date picker #}
         <input type="hidden" name="{{ field.name }}" id="{{ field.id }}-datestorage" autofocus
-               {% if field.data and not field.date_missing %}value="{{ field.data|format_date('code') }}"{% endif %}>
+               {% if field.data and not field.date_missing %}value="{{ field.data|format_date('code') }}"{% endif %}
+               {{ input_args | html_params }}>
         <span id="{{ field.id }}"></span>
     </div>
 {% endblock %}
```